ui/src/app/applications/components/application-create-panel/application-create-panel.tsx

```diff
@@ -219,12 +219,27 @@ export const ApplicationCreatePanel = (props: {
                                                             />
                                                         </div>
                                                     ) || type === 'Helm' && (
-                                                        <div className='argo-form-row'>
-                                                            <FormField formApi={api} label='Values Files' field='spec.source.helm.valueFiles' componentProps={{
-                                                                options: details.helm && details.helm.valueFiles,
-                                                                noTagsLabel: 'No values files selected',
-                                                            }} component={TagsInputField} />
-                                                        </div>
+                                                        <React.Fragment>
+                                                            <div className='argo-form-row'>
+                                                                <FormField formApi={api} label='Values Files'
+                                                                           field='spec.source.helm.valueFiles'
+                                                                           componentProps={{
+                                                                               options: details.helm && details.helm.valueFiles,
+                                                                               noTagsLabel: 'No values files selected',
+                                                                           }} component={TagsInputField}/>
+                                                            </div>
+                                                            <div className='argo-form-row'>
+                                                                <pre><FormField formApi={api} label='Values'
```

Comment:
Label inside of pre looks different. 
![image](https://user-images.githubusercontent.com/426437/62745213-bcb01980-b9fe-11e9-9ee4-a7d3d261413e.png)
Instead please use selector which changes style only of textarea.  E.g.
```html
<div className='argo-form-row application-create-panel__pre'>
```

```scss
.application-create-panel {
  &__pre textarea, pre {
      font-family: monospace;
      line-height: normal;
      white-space: pre;
  } 
}

```